core/services/gateway/handlers/functions/api.go

```diff
@@ -3,6 +3,8 @@ package functions
 import "github.com/smartcontractkit/chainlink/v2/core/services/gateway/api"
 
 const (
+	// Note: any update to this list shuold be reflected in
+	// HandlerTypeForMethod in handler_factory.go
```

Comment:
To reduce the risk of missing this in the future, we could potentially add a Methods() getter to the Handler interface. Then HandlerFactory constructor could go over each handler, collect method names, ensure they are unique and build the HandlerTypeForMethod function automatically.